src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,43 @@
+import classNames from 'classnames';
+import { Errors } from '../../types/Errors';
+import { Dispatch, SetStateAction, useEffect } from 'react';
+
+type Props = {
+  error: Errors;
+  setErrorMessage: Dispatch<SetStateAction<Errors>>;
+};
+export const ErrorNotification: React.FC<Props> = props => {
```

Comment:
1) u can use another syntax(up to u)
2) fix indents for all the cases

```suggestion
  setErrorMessage: (error: Errors) => void;
};

export const ErrorNotification: React.FC<Props> = props => {
```